client/components/foldable-card/docs/example.jsx

```diff
@@ -14,6 +14,18 @@ export default class FoldableCardExample extends PureComponent {
 					</FoldableCard>
 				</div>
 
+				<div>
+					<FoldableCard
+						header="This is a foldable card with smooth animation"
+						screenReaderText="More"
+						smooth
+						contentExpandedStyle={ { 'max-height': '70px' } }
+					>
+						<p>These are its contents</p>
+						<p>And some more</p>
```

Comment:
To get the "smooth" FoldableCard devdocs example looking ok, we should add some padding to the contents within the FoldableCard. One way, just for this example, could be like this:

```suggestion
						contentExpandedStyle={ { 'max-height': '112px' } }
					>
					    <div style={ { padding: '16px 16px 0' } }>
						    <p>These are its contents</p>
						    <p>And some more</p>
						</div>
```